_episodes/01-plone.md

```diff
@@ -3,48 +3,117 @@ title: Introduction to the Data Portal
 
-You are encouraged to read the [OTN FAQs Page](https://members.oceantrack.org/faq) for more information.
+OTN's `database` is built on PostgreSQL/PostGIS and is hosted on OTN hardware at Dalhousie University. Many partner Nodes are hosted at other locations. Users do not have direct write access to the database: the files posted in a Data Portal folder will be downloaded, quality controlled and loaded into the database by a Data Manager.
 
-### How to register with the OTN Database
+# Useful tools for Field Techs
 
-In order to register a project with OTN, we require 3 metadata types:
-1. project metadata
-2. instrument deployment metadata
-3. tagging metadata
+OTN's Data Portal website is used extensively by OTNHQ staff members for many purposes. A few of these are highlighted below!
 
-See the templates [here](https://members.oceantrack.org/data/data-collection).  OTNDC@DAL.CA is the best contact for assistance
+## Project Pages
 
-### What is the benefit of registering with the OTN Database?
+Each project which has registered with OTN or a partner Node will have a public project summary page. These are searchable from the `Home page` using the interactive map tool, or the map search option. An example project page for project NCAT is here [https://members.oceantrack.org/OTN/project?ccode=NCAT](https://members.oceantrack.org/OTN/project?ccode=NCAT).
 
-OTN and affiliated networks provide automated cross-referencing of your detection data with other tags in the system to help resolve "mystery detections" and provide detection data to taggers in other regions. OTN's Data Manager will also extensively quality control your submitted metadata for errors to ensure the most accurate records possible are stored in the database. OTN's database and Data Portal website are excellent places to archive your datasets for future use and sharing with collaborators. We offer pathways to publish your datasets with [OBIS](https://obis.org/), and via open data portals like ERDDAP, GeoServer etc. The data-product format returned by OTN is directly ingestible by analysis packages such as [glatos](https://github.com/ocean-tracking-network/glatos) and [resonATe](https://gitlab.oceantrack.org/otndc/resonate) for ease of analysis. OTN offers support for the use of these packages and tools.
+This page contains the following sections:
 
-### What is the Data Portal?
+1. TItle, citation, Node membership information and collaboration type (Tracker, Deployment or Data)
+2. An interactive map showing non-clickable tagging sites and clickable station-history information
+3. A list of the species studied by the project
+4. Contact information for Principal Investigators and Points of Contact
+5. Related URLs including:
```

Comment:
want to talk about the diffs when its a published project folder?

Reply:
can do